lagtraj/forcings/conversion/targets/sam.py

```diff
@@ -0,0 +1,631 @@
+"""This module contains:
+    1. a list of the SAM variables as a dictionary
+    2. the mapping between the internal lagtraj variables (era5) and SAM
+    3. and a function to process the conversion
+    4. a function to compute some time related variables for SAM.
+    Notes: this module is created based on kpt.py because Peter Blossey's matlab conversion scripts
+           take in kpt format output.
+"""
+import datetime
+
+import numpy as np
+import scipy.interpolate as intp
+import xarray as xr
+
+# from ....domain.sources.era5.constants import rg
+from ....utils.interpolation.methods import central_estimate, steffen_1d_no_ep_time
+
+# following Peter Blossey's matlab script structure:
+sam_attributes = {
+    "Ps": {"units": "Pa", "long_name": "surface pressure"},
+    "Ptend": {"units": "Pa/s", "long_name": "surface pressure tendency"},
+    "Tg": {
+        "units": "K",
+        "long_name": "Surface (skin) Temperature (SST if over water)",
+        "t_skin_correct": "Skin temperature has been corrected by 1.000000. "
```

Comment:
We are working to remove this correction, and just using the ERA5 skin temperature, for the MONC runs in a new pull request.

Reply:
(this has been implemented for 0.1.0)

Reply:
#168

Reply:
I just updated my local repo to match v0.1.0. :tada:
It seems that the [attribute for t_skin in kpt.py](https://github.com/EUREC4A-UK/lagtraj/blob/a0dac9d707b07692a1924af7096c027c2b35493f/lagtraj/forcings/conversion/targets/kpt.py#L131) is still saying t_skin has been corrected.  Maybe the attribute there will also need to be updated?

Reply:
You are right, I will open an issue!